### ISSUE TITLE
Algokit Verification Not Working

### ISSUE THREAD

**Issue by msbritneymariela** [algorandfoundation/algokit-cli#243]
I have just installed all of the required prequesites to start using Algokit. However, I ran into a bug. 

However, when trying to verify that it was installed using the following code:

algokit --version

I am receiving the following error message.

algokit : The term 'algokit' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ algokit version
+ ~~~~~~~
    + CategoryInfo          : ObjectNotFound: (algokit:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

This same message also shows up when trying to run: algokit doctor. 

I have tried restarting the visual studio code app, my computer, and even tried using new terminals, but none of them worked to resolve this issue.

But ccording to Visual Studio Code, Algokit is installed, but the code to verify it is refusing to work and is telling me that the code does not exist, so I cannot verify if it was successfull installed or not, nor can I repair it because to the code provided to fix any probems if there are problems is giving the same error.

Long story short, Algokit is installed on my Windows computer, but the code to verify that it was installed correctly does not work at all.

**Comment by barnjamin** [comment]
Its likely that the executable is not on your PATH, so its not a recognized command. It _should_ automatically happen when you install it.

You could try `pipx ensure path` 

Or manually add it to your path, on windows I think the executable is in `C:\Users\$username\.local\bin`

**Comment by daniel-makerx** [comment]
This is definitely an issue of `algokit` not being found on the path, as @barnjamin mentioned it should be resolvable with `pipx ensurepath` or possibly `python -m pipx ensurepath` if `pipx` is also not found on the path. I have added a note to the verification step about this (see #248).
